<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerSettingsPythonPanel</class>
 <widget class="ctkSettingsPanel" name="qSlicerSettingsPythonPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Python</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="GeneralGroupBox">
     <property name="title">
      <string>General</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="EditorLabel">
        <property name="text">
         <string>Editor for .py files:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ctkPathLineEdit" name="EditorPathLineEdit">
        <property name="toolTip">
         <string>Select an executable for editing .py files. If left empty then the default program associated with .py files will be launched.</string>
        </property>
        <property name="settingKey">
         <string>PythonEditor</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ShellDisplayGroupBox">
     <property name="title">
      <string>Python Console</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="DockableWindowLabel">
        <property name="text">
         <string>Display in dockable window:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ctkCheckBox" name="DockableWindowCheckBox">
        <property name="toolTip">
         <string>Display Python console in a window that can be placed inside the main window. If disabled then the Python Console is displayed as an independent window.</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="PromptFontLabel">
        <property name="text">
         <string>Font:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ctkFontButton" name="ConsoleFontButton">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="LogLevelLabel">
        <property name="text">
         <string>Log level:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="ConsoleLogLevelComboBox">
        <property name="toolTip">
         <string>Log messages at this level and above are displayed in the Python console.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCheckBox</class>
   <extends>QCheckBox</extends>
   <header>ctkCheckBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkFontButton</class>
   <extends>QPushButton</extends>
   <header>ctkFontButton.h</header>
  </customwidget>
  <customwidget>
   <class>ctkPathLineEdit</class>
   <extends>QWidget</extends>
   <header>ctkPathLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSettingsPanel</class>
   <extends>QWidget</extends>
   <header>ctkSettingsPanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
